src/uidgen.c

```diff
@@ -1,3 +1,9 @@
+#define _BSDSOURCE
+#define __USE_BSD
```

Comment:
http://stackoverflow.com/questions/19525378/be64toh-not-linking-or-being-declared-when-compiling-with-std-c99